Makefile

```diff
@@ -45,3 +45,12 @@ check: checkstatic unittests
 
 testdata:
 	php tools/raisinbread_refresh.php
+
+docker-install:
+	if [ ! -d "smarty/templates_c" ]; then mkdir smarty/templates_c; fi
+	if [ ! -d "uploadedfiles" ]; then mkdir uploadedfiles; fi
+	chmod 770 smarty/templates_c
+	chmod 770 uploadedfiles
```

Comment:
I don't see why building the docker image should change things in the non-docker code ase

Reply:
It is not necessary, it has been removed.